cfc-testing/scripts/compliance-report-generator.py

```
@@ -0,0 +1,1389 @@
+                        config_name = match.group(1)
+
+                if config_name and config_name in results_by_config:
+                    # Update duration for this config
+                    results_by_config[config_name]['duration'] = time_val
+
+            # Rewrite the incremental file with updated durations
+            with open(self.incremental_results_file, 'w') as f:
+                for result in results_by_config.values():
+                    f.write(json.dumps(result) + '\n')
+
+        except Exception as e:
```

Ruff (0.14.8)

Do not catch blind exception: `Exception`

(BLE001)